config/routes.rb

```diff
@@ -8,15 +8,15 @@
   Hydra::BatchEdit.add_routes(self)
   # This must be the very last route in the file because it has a catch-all route for 404 errors.
     # This behavior seems to show up only in production mode.
-  mount Sufia::Engine => '/'
+  mount Sufia::Engine => '/admin'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.